runtime/cluster.cu

```diff
@@ -0,0 +1,93 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#if (defined(__CUDA_ARCH__) && (__CUDA_ARCH__ >= 900))
+
+namespace Hopper {
```

Comment:
nit: Do we really need this namespace?

Reply:
I copied what was done for the tma instructions in runtime/memory.cpp. The cluster operations are restricted to hopper+.

Reply:
They are also forward compatible, i.e., Blackwell has the same API, right? I'd agree this namespace would appear more appropriate if they are sm_90a.

Reply:
I removed `hopper` namespace.